timescaledb/tutorials/nfl-fantasy-league.md

```diff
@@ -174,19 +174,249 @@ conn.close()
 Now that you have all the data ingested, let's go over some ideas on how you can analyze the data to help you perfect
 your fantasy drafting strategy and win your fantasy season.
 
-To analyze the dataset you will need to install two libraries, if you haven't already: Pandas and Matplotlib.
+To analyze the dataset you will need to install two libraries, if you haven't already: Pandas, Matplotlib, and Numpy.
 
-### Install pandas and matplotlib
+### Install pandas, matplotlib, and numpy
 ```bash
-pip install pandas matplotlib
+pip install pandas matplotlib numpy
 ```
 
 1. Number of yards run in game for passing plays, by player and game 
 1. Avg yards run for a player over a game
 1. Average and median yards run per game by type of player (not taking avg of individual)
 1. Num of snap plays by player where they were on the offense
-1. Number of plays vs points scored
-1. Average yards per game for top 3 players of each position
+1. [Number of plays vs points scored](number-of-players-vs-points-scored)
+1. [Average yards per game for top 3 players of each position](#average-yards-per-game-for-top-3-players-of-each-position)
+
+### **Number of plays vs points scored**
+
+We will be using the following query to get data on the number of plays and final score for each game during the 2018 season. This data will be separated by team so that we can compare the number of plays with a team's win or loss.
```

Comment:
```suggestion
Use this query to get data on the number of plays and final score for each game during the 2018 season. This data is separated by team so that we can compare the number of plays with a team's win or loss.
```